packages/getting-started/src/browser/getting-started-widget.tsx

```diff
@@ -224,6 +234,7 @@ export class GettingStartedWidget extends ReactWidget {
 
         return <div className='gs-section'>
             <h3 className='gs-section-header'><i className={codicon('folder-opened')}></i>{nls.localizeByDefault('Open')}</h3>
```

Comment:
I wonder whether we should also call this section 'Start' instead of 'Open' as it is in VS Code as this command does not really reflect opening anything. What do you think?

Reply:
I think it is a good idea.